### PR TITLE
update pybedtools 0.6.9

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

File: packages/package_python_2_7_pybedtools_0_7_4/tool_dependencies.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    </package>
+    <package name="samtools" version="0.1.19">
+        <repository name="package_samtools_0_1_19" owner="iuc" />
+    </package>
+    <package name="bedtools" version="2.24">
+        <repository name="package_bedtools_2_24" owner="iuc" />
+    </package>
+    <package name="cython" version="0.20.1">
+        <repository name="package_cython_0_20_1" owner="iuc" prior_installation_required="True" />
+    </package>
```
Comment (yhoogstrate) [inline packages/package_python_2_7_pybedtools_0_7_4/tool_dependencies.xml]: Is this missing?:

```
    <package name="python" version="2.7.10">
        <repository name="package_python_2_7_10" owner="iuc" prior_installation_required="True" />
    </package>
```

File: packages/package_python_2_7_pybedtools_0_6_9/tool_dependencies.xml [hunk 2]
```diff
@@ -25,6 +25,9 @@
                     <repository name="package_cython_0_20_1" owner="iuc">
                         <package name="cython" version="0.20.1" />
                     </repository>
+                    <repository name="package_bedtools_2_22" owner="iuc">
```
Comment (yhoogstrate) [inline packages/package_python_2_7_pybedtools_0_6_9/tool_dependencies.xml]: 2.24?

File: packages/package_python_2_7_pybedtools_0_7_4/.shed.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+- Tool Dependency Packages
+description: Contains a tool dependency definition that downloads and compiles version
+  0.7.4 of the pybedtools package.
+long_description: |
+  pybedtools is a Python wrapper for Aaron Quinlan's BEDtools programs
+  (https://github.com/arq5x/bedtools), which are widely used for genomic interval
+  manipulation or "genome algebra". pybedtools extends BEDTools by offering feature-level
+  manipulations from with Python. See full online documentation, including installation
+  instructions, at http://pythonhosted.org/pybedtools/.
+name: package_python_2_7_pybedtools_0_7_4
```
Comment (yhoogstrate) [inline packages/package_python_2_7_pybedtools_0_7_4/.shed.yml]: Should it be named package_python_2_7_pybedtools_0_7_4 or package_python_2_7_10_pybedtools_0_7_4?

File: packages/package_python_2_7_pybedtools_0_7_4/tool_dependencies.xml [hunk 4]
```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <package name="tabix" version="0.2.6">
+        <repository name="package_tabix_0_2_6" owner="iuc" prior_installation_required="True" />
+    </package>
+    <package name="samtools" version="0.1.19">
```
Comment (yhoogstrate) [inline packages/package_python_2_7_pybedtools_0_7_4/tool_dependencies.xml]: Is this dependency necessary for the package (it is not required for installation) or is it needed by tools depending on this?

File: packages/package_python_2_7_pybedtools_0_6_9/tool_dependencies.xml (hunk 2 above)
Comment (bgruening) [inline packages/package_python_2_7_pybedtools_0_6_9/tool_dependencies.xml]: It already depends on this old version, just wasn't sourced. So I would like to keep it in this version.

File: packages/package_python_2_7_pybedtools_0_7_4/.shed.yml (hunk 3 above)
Comment (bgruening) [inline packages/package_python_2_7_pybedtools_0_7_4/.shed.yml]: I'm not sure. My hope as I started is that we only have one python_2_7 package that we constantly upgrade. Python 2.7 should be ABI compatible. What do you think, do we want several python packages or only one?

File: packages/package_python_2_7_pybedtools_0_7_4/tool_dependencies.xml (hunk 1 above)
Comment (bgruening) [inline packages/package_python_2_7_pybedtools_0_7_4/tool_dependencies.xml]: Fixed thanks!

**Comment by bgruening** [comment]
Thanks @yhoogstrate!

File: packages/package_python_2_7_pybedtools_0_6_9/tool_dependencies.xml (hunk 2 above)
Comment (yhoogstrate) [inline packages/package_python_2_7_pybedtools_0_6_9/tool_dependencies.xml]: Great!

File: packages/package_python_2_7_pybedtools_0_7_4/.shed.yml (hunk 3 above)
Comment (yhoogstrate) [inline packages/package_python_2_7_pybedtools_0_7_4/.shed.yml]: > do we want several python packages or only one

I think having multiple packages only complicates maintainance. As long as the python packages are compiled with the same settings it should be fine. Thanks! I'll cross check if installation works and make sure we can merge it asap.

**Comment by yhoogstrate** [comment]
I have updated 0.7.4 in the testtoolshed. Both versions install fine over here so I think it's ready to be merged. Thanks @bgruening

File: packages/package_python_2_7_pybedtools_0_7_4/.shed.yml (hunk 3 above)
Comment (bgruening) [inline packages/package_python_2_7_pybedtools_0_7_4/.shed.yml]: Thanks!

**Pull request merged**